### PR TITLE
Change deprecated nibabel function get_data to get_fdata

### PR DESCRIPTION
**Pull request by MatthieuJoulot**
Changed the nibabel functions get_data() to get_fdata() since get_data() will soon no longer be available.

### CODE REVIEW

File: clinica/iotools/converters/adni_to_bids/adni_modalities/adni_t1.py [hunk 1]
```diff
@@ -180,10 +181,10 @@ def compute_t1_paths(source_dir, csv_dir, dest_dir, subjs_list, conversion_dir):
         # ('022_S_5004', 'bl'),
         # ('022_S_5004', 'm03'),
         # Several images: T1wa ...
-        ("006_S_4485", "m84"),
-        ("029_S_2395", "m72"),
-        ("114_S_6039", "bl"),
-        ("016_S_4952", "m48"),
+        #("006_S_4485", "m84"),
```
Comment (omar-rifai) [inline clinica/iotools/converters/adni_to_bids/adni_modalities/adni_t1.py:184]: I think these changes need to be in a separate PR as they do not address a problem related to the the changes in the nibabel function

File: clinica/pipelines/machine_learning_spatial_svm/spatial_svm_utils.py [hunk 2]
```diff
@@ -1086,7 +1086,7 @@ def heat_solver_equation(input_image, g, FWHM, t_step, dartel_input):
     beta = sigma ** 2 / 2
 
     input_image_read = nib.load(input_image)
-    input_image_data = input_image_read.get_fdata()
+    input_image_data = input_image_read.get_fdata(dtype="float32")
     input_image_data = np.array(input_image_data, dtype="float32")
```
Comment (ghisvail) [inline clinica/pipelines/machine_learning_spatial_svm/spatial_svm_utils.py:1090]: On doit pouvoir se débarrasser de cette ligne de conversion en `np.array` du coup.

File: clinica/pipelines/pet_surface/pet_surface_utils.py [hunk 3]
```diff
@@ -178,7 +178,7 @@ def isclose(a, b, rel_tol=1e-9, abs_tol=0.0):
     # Read label from gtmsegfile, change data into integers in order to have no problems when testing equality of labels
     label = nib.load(gtmsegfile)
     label.header.set_data_dtype("int8")
-    volume = label.get_fdata()
+    volume = label.get_fdata(dtype="float32")
```
Comment (ghisvail) [inline clinica/pipelines/pet_surface/pet_surface_utils.py]: There is something weird going on here. The comment says we want to load the data as integers, yet we load the volume as float.

**Comment by omar-rifai** [comment]
Thanks @MatthieuJoulot, this looks good. I'll merge
